src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1932,8 +1821,25 @@ private static TypeSymbolWithAnnotations InferResultNullability(Conversion conve
                     var methodOpt = conversion.Method;
                     if ((object)methodOpt != null && methodOpt.ParameterCount == 1)
                     {
+                        if (Binder.RequiresLiftedUserDefinedConversion(methodOpt, operandType.TypeSymbol))
```

Comment:
>Binder.RequiresLiftedUserDefinedConversion(methodOpt, operandType.TypeSymbol) [](start = 28, length = 77)

Is lifted flag not sufficient on the conversion node?

Reply:
Changed to `if (conversion.BestUserDefinedConversionAnalysis.Kind == UserDefinedConversionAnalysisKind.ApplicableInLiftedForm)`.